src/views/Hearing/Sidebar.js

```diff
@@ -35,6 +37,38 @@ class Sidebar extends React.Component {
     );
   }
 
+  getLanguageChanger() {
+    const {hearing, activeLanguage} = this.props;
+    const availableLanguages = { fi: 'Kuuleminen Suomeksi', sv: 'Enkäten på svenska', en: 'Questionnaire in English'};
+    const languageOptionsArray = config.languages.map((lang) => {
+      if (getAttr(hearing.title, lang, {exact: true}) && lang === activeLanguage) {
```

Comment:
Also, if UI language is Swedish and the Hearing is available only in Finnish and English, none of the languages are highlighted.

The default hearing language (Finnish in this case) should be highlighted even if the UI is in another language.

Essentially, the sidebar language list should always display which hearing language we are looking at and what are the other available languages, regardless of the language the UI is in.